<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HGradeItemDialog</class>
 <widget class="QDialog" name="HGradeItemDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>分级数据编辑</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="HEntireTableWidget" name="tableWidget">
     <property name="editTriggers">
      <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>100</number>
     </attribute>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_01">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_01">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>优先级：</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>spinBox_01</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_01"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_02">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_02">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>分级数：</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>spinBox_02</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_02"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_03">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_01">
       <property name="text">
        <string>平均分级</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_01">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>计 算</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_09">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_11">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_11">
       <property name="text">
        <string>确 定</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_13">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_12">
       <property name="text">
        <string>取 消</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_19">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HEntireTableWidget</class>
   <extends>QTableWidget</extends>
   <header>HePlugin/HEntireTableWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_11</sender>
   <signal>clicked()</signal>
   <receiver>HGradeItemDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>326</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_12</sender>
   <signal>clicked()</signal>
   <receiver>HGradeItemDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>323</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>327</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
